src/components/MoneyRequestConfirmationList.js

```diff
@@ -169,7 +172,17 @@ function MoneyRequestConfirmationList(props) {
             });
         }
         return sections;
-    }, [selectedParticipants, getParticipantsWithAmount, props.hasMultipleParticipants, props.iouAmount, props.iouCurrencyCode, props.participants, translate, payeePersonalDetails]);
+    }, [
+        props.participants,
+        props.hasMultipleParticipants,
+        props.iouAmount,
+        props.iouCurrencyCode,
+        getParticipantsWithAmount,
+        selectedParticipants,
+        payeePersonalDetails,
+        translate,
```

Comment:
Not sure why we need `translate` here.

NAB (as it's not added here)

Reply:
Oh interesting, thanks for calling that out. I was previously under the impression that including `translate` in the dependency array was in case the function changed, not what the function returns 🙃